### ISSUE TITLE
Unicode correction and font conversion

### ISSUE THREAD

**Issue by rugk** [rugk/awesome-emoji-picker#106]
## Background

Many features like Unicode smart quotes („“, “”) are hard to use/notice, because no keyboard has them.
I personally e.g. often search Wikipedia for the Unicode symbol and copy it.
**Aim:** We want a simpler solution for that.

## Proposed solution

* make a separate add-on with the template: https://github.com/TinyWebEx/AddonTemplate
* create all the marketing stuff around it (we need a funny/proper name, Readme, AMO description etc.)
* As @tdulcet did the main work, they can be co-author etc.
* **Additional idea:** We can integrate some "advertisement" between both add-ons, so one gets to know the other add-on when only one is installed. On the settings page and as a [tip](https://github.com/TinyWebEx/RandomTips).
  This way, we adress the problem from https://github.com/rugk/awesome-emoji-picker/pull/93#issuecomment-702672127:
  > I thought it would be nice to have one add-on handle everything related to Unicode and Unicode autocorrections. I doubt users would install one add-on for Emoji autocorrect, one for Unicode smart quotes autocorrect and another one for Unicode fractions autocorrect. In addition, most users probably do not know the difference between Unicode symbols and emojis, considering that most emojis are also symbols.

## Alternatives

Integrate this in this add-on (see https://github.com/rugk/awesome-emoji-picker/pull/93). I do consider this out-of-scope of this project, however.
But I do like the idea.

**Comment by rugk** [comment]
So from #93 I would like to split these features into the new add-on:
![unicodesplit](https://user-images.githubusercontent.com/11966684/96255485-4706ec80-0fb7-11eb-8815-4dfcb40531a5.png)

**Comment by tdulcet** [comment]
@rugk How do you want to move forward with this? Do you want to create the new repository or would you like me to do so? Now that I am removing the three options from #93, I would like to put them somewhere.

For anyone following along, there is a demo of all six options for autocorrecting Unicode symbols/emojis in #66, including the three options that will be included in the new add-on. There is also a version of this add-on [here](https://tealdulcet.com/awesome-emoji-picker.zip) for testing, which the above and below screenshots were taken of. It includes:
1. The Awesome Emoji Picker (this add-on)
2. The [six options](https://github.com/rugk/awesome-emoji-picker/issues/66#issuecomment-559033052) for autocorrecting Unicode symbols/emojis (three for #93 and three for the new add-on)
3. Full Thunderbird support (#49, for a future PR and the new add-on)
    1. Blocked by https://github.com/rugk/awesome-emoji-picker/issues/49#issuecomment-635390352
4. Full Chrome support (#54, for a future PR and the new add-on)
    1. Blocked by https://github.com/rugk/awesome-emoji-picker/issues/4#issuecomment-703043742
5. Uses the actual native emoji for the browser action icon (for a future PR and possibly the new add-on)
    1. Requires that the user enabled the color icon and native emojis
6. [Unicode font conversion](https://github.com/rugk/awesome-emoji-picker/pull/93#issuecomment-710780555) (for the new add-on)

Here is a screenshot of the Unicode font conversion feature in Firefox: ![image](https://user-images.githubusercontent.com/15305150/96453020-c02c5c80-11ce-11eb-8ce2-5e10446d3c40.png)

and in Chrome: ![image](https://user-images.githubusercontent.com/15305150/96453048-cb7f8800-11ce-11eb-864a-06f48de6f2ef.png)

**Comment by rugk** [comment]
> How do you want to move forward with this? Do you want to create the new repository or would you like me to do so? Now that I am removing the three options from #93, I would like to put them somewhere.

I am fine with anything.
But here you go, if you want: https://github.com/TinyWebEx/unicodify I've invited you as a maintainer. :blush: 

It's really just the basic template, did not had much time to change much else.

**Comment by tdulcet** [comment]
> But here you go, if you want: https://github.com/rugk/unicodify I've invited you as a maintainer. 😊

Thanks!

> It's really just the basic template, did not had much time to change much else.

OK, I will get started on a PR at the new repository with these features.

**Comment by tdulcet** [comment]
@rugk Sorry to bother you with this. I tried to accept your invitation when you first sent it, but I just get this 500 server error: 
![image](https://user-images.githubusercontent.com/15305150/97192536-38fa5e00-1765-11eb-9fbb-73766f5fff64.png)

After a day of that error I contacted GitHub Support and I just finally got this response after another six days:
```
Hi Teal,

Is it possible to have the repo or organisation owner to cancel and resend the invitation?

Please have them try this and let us know if you still see errors.

Thanks,

Robert
```

I am not sure why they will not just fix their issue or at least accept your invitation for me... 😬 Anyway, this is why I have not made any progress yet on the PR.

**Comment by rugk** [comment]
Okay invited again, [you should now be able to accept that](https://github.com/rugk/unicodify/invitations). Strange thing, but thanks for keeping me up-to-date… :smiley:

**Comment by tdulcet** [comment]
Thanks, I was able to accept your invitation this time! 🎉 Sorry again about the hassle. I will get started on that PR now...
